src/main/resources/fxml/pkt_capture/Layout.fxml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import com.cisco.trex.stl.gui.controllers.Record?>
+<?import com.cisco.trex.stl.gui.controllers.Monitor?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.TabPane?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.control.Tab?>
+<?import javafx.scene.layout.AnchorPane?>
+<BorderPane xmlns="http://javafx.com/javafx/8"
```

Comment:
Initially I have another idea of whole layout design. Then it changed, but root layout is left.
Fixed.